packages/app/src/app/pages/common/Modals/Common/Alert.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Element, Button, Text, Stack } from '@codesandbox/components';
+import css from '@styled-system/css';
+
+type Props = {
+  onCancel?: () => void;
+  onPrimaryAction?: () => void;
+  children?: any;
```

Comment:
```suggestion
```

Reply:
I need the children here for showing other things

Reply:
It's included if you use `FunctionComponent`